### ISSUE TITLE
Request: Auto-save unsaved tabs every _ minutes.

### ISSUE THREAD

**Issue by Te-M** [josh-berry/tab-stash#224]
I know it's basically just profiles but I much prefer how easy it is to prune tabs using Tab Stash.  I use a hardened version of Firefox and unfortunately it uses No History mode which means once a window is closed it's gone.  Since I run into the occasional crash or accidental close this would be a useful feature for me personally.  It could even sort it into a "closed tab" subfolder to help keep it all sorted well.  Thanks!

**Comment by desbest** [comment]
[Use the Tab Session Manager extension](https://addons.mozilla.org/en-US/firefox/addon/tab-session-manager/)

**Comment by Te-M** [comment]
> [Use the Tab Session Manager extension](https://addons.mozilla.org/en-US/firefox/addon/tab-session-manager/)
Unfortunately that extension isn't supported in that mode.

**Comment by Te-M** [comment]
Yep, definitely a pickle, I agree!  For me I'd be okay with it saving only tabs that were open for a long time in a group it would eventually purge awhile after the last close.  But I'm guessing the real answer is just to change my user behavior lol.

**Comment by follyx** [comment]
I am feeling pretty well with your extension, but...
If I collapse a saved tab the headline isnst readable anymore. Suggest you have a lot reusable saved tabs and look forward to find a specific one, I always have to open or let got the all the tabs. 
Isnt the readability from saved tabs important? ;-)
Hopefully you think about to implement this in the near future.

Keep going.

**Comment by Te-M** [comment]
Oh!  I should probably mention I did find an extension that works with a hardened Firefox, it's called Save My Tabs!.  It's not nearly as complex and it requires curation because you can really only make it autosave **everything** every so many minutes, but I use it in addition with Tab Stash just in case crashes happen.

**Comment by Snake883** [comment]
It would be nice if the "Unstashed Tabs" is saved every X minutes.
